### PR TITLE
CHANGELOG: Add updates for version 7

### PR DESCRIPTION
**Pull request by scorphus**
# Description

Prepare files for the next release, v7.

# Checklist:

- [ ] ~My code follows the [style guidelines](https://github.com/oh-my-fish/oh-my-fish/blob/master/CONTRIBUTING.md#code-style) of this project~
- [x] I have performed a self-review of my own code
- [ ] ~I have commented my code, particularly in hard-to-understand areas~
- [x] I have made corresponding changes to the documentation
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,51 @@
 Important changes to Oh My Fish are recorded here for each release.
 
 
+## [7] - 2020-04-10
+
+Another release that took long but is finally out! v7 support Fish 3.0 and 3.1 and brings a few improvements and bug fixes.
```
Comment (sagebind) [inline CHANGELOG.md:7]: Small grammar suggestion.

```suggestion
Another release that took long but is finally out! v7 supports Fish 3.0 and 3.1 and brings a few improvements and bug fixes.
```

Comment (scorphus) [reply CHANGELOG.md]: Thanks for reviewing and spotting that one! Fixed 🙌

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,51 @@
...
+- [Anton-Latukha](anton.latukha@gmail.com)
+- [Benedict Lee](benedictleejh@gmail.com)
+- [Dalton Duvio](thebigdalt@gmail.com)
+- [Édouard Lopez](edouard-lopez@users.noreply.github.com)
+- [Fabian Homborg](FHomborg@gmail.com)
+- [Frederick Henderson](frederickjh@users.noreply.github.com)
+- [Gabriel César](gcal1@live.com)
+- [Jan T. Sott](jan@idleberg.com)
+- [Joel Burget](joelburget@gmail.com)
+- [Justin Hileman](mailto:github@0x7f.us)
```
Comment (bobthecow) [inline CHANGELOG.md:35]: should all the rest have a `mailto:` or this one not?

Comment (scorphus) [reply CHANGELOG.md:36]: Good one too, Justin! Thanks for reviewing! Fixed.

I went over the list of contributors to this release and fixed it as well. Thanks for triggering that.

One last review?

Thanks in advance!

**Pull request merged**